Update README.md (#916)
veggiemonk/awesome-docker: README.md

### User Interface

> #### IDE integrations

Reviewer: I think it's a good idea to have a section for IDEs here, but it could be highly opinionated (but so is this list).
What do you think @veggiemonk / @agebhar1?

Author: Well, you may wonder how many peoples don't even thought that their IDE can work with Docker.
I created a separate commit with IDE integrations so you may cherry pick only the first commit.

Reviewer 2: Good idea! 👍

- JetBrains IDEs (IntelliJ IDEA, GoLand, WebStorm, CLion etc.) has [built-in Docker plugin](https://www.jetbrains.com/help/idea/docker.html#managing-images)
- Eclipse [Docker Tooling plugin](https://www.eclipse.org/community/eclipse_newsletter/2016/july/article2.php)

#### Desktop

Native desktop applications for managing and montoring docker hosts and clusters

- [Captain](https://getcaptain.co/) - Manage containers from the MacOSX menu bar by [@RickWong](https://github.com/rickwong)
- [Dockeron](https://github.com/dockeron/dockeron) - A project built on Electron + Vue.js for Docker on desktop. [@fluency03](https://github.com/fluency03)

> - [Docker Desktop](https://www.docker.com/products/docker-desktop) - Official native app. Only for Windows and MacOS

Reviewer: Nice, how could we not have this on our list already... 😅

- [ctop (2)](https://github.com/bcicen/ctop) - Top-like interface for container metrics (Golang) by [@bcicen](https://github.com/bcicen/)
- [dext-docker-registry-plugin](https://github.com/vutran/dext-docker-registry-plugin) - Search the Docker Registry with the Dext smart launcher.
- [dive](https://github.com/wagoodman/dive) - A tool for exploring each layer in a docker image. By [wagoodman](https://github.com/wagoodman).

> - [DockDash](https://github.com/byrnedo/dockdash) detailed stats. By [byrnedo](https://github.com/byrnedo).

Reviewer: ```suggestion
- [Dockdash](https://github.com/byrnedo/dockdash) detailed stats. By [byrnedo](https://github.com/byrnedo).
```

- [dockersql](https://github.com/crosbymichael/dockersql) - A command line interface to query Docker using SQL by [@crosbymichael](https://github.com/crosbymichael)
- [DockSTARTer](https://github.com/GhostWriters/DockSTARTer) - DockSTARTer helps you get started with home server apps running in Docker by ([GhostWriters](https://github.com/GhostWriters))
- [dockly](https://github.com/lirantal/dockly) - An interactive shell UI for managing Docker containers by [@lirantal](https://github.com/lirantal)

> - [DocUI](https://github.com/skanehira/docui) - An interactive shell UI for managing Docker containers. Also works in Windows. By [@skanehira](https://github.com/skanehira)

Reviewer: Nice addition, i'd suggest keeping the title in the same casing as mentioned in the source for consistency reasons

```suggestion
- [docui](https://github.com/skanehira/docui) - An interactive shell UI for managing Docker containers. Also works in Windows. By [@skanehira](https://github.com/skanehira)
```

- [docker.vim](https://github.com/skanehira/docker.vim) - Manage docker containers and images in Vim. By [@skanehira](https://github.com/skanehira)
- [dry](https://github.com/moncho/dry) - An interactive CLI for Docker containers by [@moncho](https://github.com/moncho)
- [DVM](https://github.com/howtowhale/dvm) - Docker version manager by [@howtowhale](https://github.com/howtowhale)
- [goinside](https://github.com/iamsoorena/goinside) - Get inside a running docker container easily. by [@iamsoorena](https://github.com/iamsoorena)

> - [LazyDocker](https://github.com/jesseduffield/lazydocker) - The lazier way to manage everything docker. A simple terminal UI for both docker and docker-compose, written in Go with the gocui library. By [@jesseduffield](https://github.com/jesseduffield)

Reviewer: i'd either keep it lowercase, maybe uppercase first, but camelcase is nowhere to be seen in the linked repo https://github.com/jesseduffield/lazydocker

```suggestion
- [lazydocker](https://github.com/jesseduffield/lazydocker) - The lazier way to manage everything docker. A simple terminal UI for both docker and docker-compose, written in Go with the gocui library. By [@jesseduffield](https://github.com/jesseduffield)
```

#### Web

> - [Admiral](https://vmware.github.io/admiral/) Web UI. By VMWare

Reviewer: Please use the link and description of the repo here:

```suggestion
- [Admiral](https://github.com/vmware/admiral) Admiral™ is a highly scalable and very lightweight Container Management platform for deploying and managing container based applications. By [VMWare]
```

VMWare has to be moved to the links section on the bottom of the file due to duplication in Registry => Harbor
